board/safety/safety_gm_passive.h

```diff
@@ -0,0 +1,22 @@
+// All sending is disallowed.
+// The only difference from "no output" model
+// is using GM ignition hook.
+
+static void gm_passive_rx_hook(CAN_FIFOMailBox_TypeDef *to_push) {
+  int bus_number = GET_BUS(to_push);
+  int addr = GET_ADDR(to_push);
+
+  if ((addr == 0x1F1) && (bus_number == 0)) {
+    bool ign = (GET_BYTE(to_push, 0) & 0x20) != 0;
+    gm_ignition_started = ign;
+  }
+}
+
+const safety_hooks gm_passive_hooks = {
+  .init = gm_init,
+  .rx = gm_passive_rx_hook,
```

Comment:
I think it's ok to just use `gm_rx_hook` here. As long as nooutput_tx_hook and default_fwd_hook are set, there is really no risk.

Reply:
@rbiasini, actually `gm_rx_hook` modifies `contorls_allowed`, and that goes into both health packet, and controls green LED of the panda. I thought it would be misleading to have a wrong value for that, hence copy-pasted code for determining the ignition.